src/AnalysisServices/AnalysisServices.Test/InMemoryTests/DataPlaneCommandTests.cs

```diff
@@ -61,7 +61,7 @@ public class DataPlaneCommandTests : AsTestsBase
                                         + "65kxhZWVUbTHaPuEvg03ZQ3esDb6wxQewJPAL-GARg6S9wIN776Esw8-53AWhzFu0fIut-9FXGma6jV7"
                                         + "MYPoUUcFuQzLZgphecPyMPXSVhummVCdBwX9sizxnmFA";
 
-        public DataPlaneCommandTests(ITestOutputHelper output)
+        public DataPlaneCommandTests(ITestOutputHelper output) : base(output)
```

Comment:
File name contains Data Plane, and the file doesn't look like a TestController, so I don't think the file needs to be edited.

Reply:
Sorry I didn't notice that AsTestsBase class had been removed.